examples/series-webgl-point/index.js

```diff
@@ -0,0 +1,61 @@
+const data = fc.randomGeometricBrownianMotion().steps(1e6)(1);
```

Comment:
I think having 10,000 points creates more consistent results for our machines. At higher values I was struggling to get any consistency